src/_helpers/sortableSet.ts

```diff
@@ -24,8 +24,26 @@ export interface Comparable {
   compare: (other: any) => number
 }
 
-export abstract class SortableSet<TItem extends Comparable> extends Set<TItem> implements Sortable<TItem> {
+export abstract class SortableSet<TItem extends Comparable> extends Set<TItem> implements Comparable, Sortable<TItem> {
   sorted (): TItem[] {
     return Array.from(this).sort((a, b) => a.compare(b))
   }
+
+  compare (other: SortableSet<TItem>): number {
```

Comment:
:question: do I want to keep this? 
Need to check if it actually benefits the code base.

Reply:
became obsolete via #165